<?xml version="1.0"?>
<launch>
  <node name="fawkes_navgraph" pkg="rcll_fawkes_sim"
        type="fawkes_navgraph" respawn="true" output="screen">
	  <param name="navgraph_file" value="$(optenv NAVGRAPH_FILE /tmp/navgraph-generated.yaml)" />
    <param name="base_frame_id" value="$(optenv BASE_FRAME_ID base_link)" />
    <param name="global_frame_id" value="$(optenv GLOBAL_FRAME_ID map)" />
  </node>
</launch>
